app/enterprise/k8s-changelog.md

```diff
@@ -2,20 +2,42 @@
 title: Kong Enterprise k8s Changelog
 ---
 
+## 1.4.2.0
+
+> Released on 2020/01/02
+
+### Summary
+
+Kong Enterprise k8s 1.4.2.0 is based on Kong Gateway 1.4.2.
+
+### Kong Gateway
+
+Kong Enterprise k8s 1.4.2.0 is based on Kong Gateway 1.4.2,
+meaning it inherits all new features and changes between Kong Gateway 1.3.0 and
+1.4.2.
+Please checkout changelog for
+[Kong Gateway 1.4.0](https://github.com/Kong/kong/blob/master/CHANGELOG.md#140).
+
+### Enterprise plugins
+
+- `upstream-tls` plugin has been dropped. Users are advised to use
```

Comment:
It was never part of K4K8 release. 

 https://github.com/Kong/kong-distributions/blob/release/kic-1.3.0.0/kong-images/kic/distributions_constants.lua

Reply:
Correct. It's mentioned in the main Enterprise 1.3.0.0 changelog:

https://github.com/Kong/docs.konghq.com/blame/master/app/enterprise/changelog.md#L306